### ISSUE TITLE
"frappe" not in list during get_installed_apps

### ISSUE THREAD

**Issue by dj12djdjs** [frappe/bench#1324]
Anytime I leave my bench for some time and I pull the latest changes then run `bench setup requirements --python` I always get this message.
```
ERROR: list.remove(x): x not in list
Traceback (most recent call last):
  File "/usr/local/bin/bench", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.8/dist-packages/bench/cli.py", line 121, in cli
    raise e
  File "/usr/local/lib/python3.8/dist-packages/bench/cli.py", line 111, in cli
    bench_command()
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/bench/commands/setup.py", line 158, in setup_requirements
    bench.setup.python(apps=apps)
  File "/usr/local/lib/python3.8/dist-packages/bench/bench.py", line 460, in python
    apps = self.bench.get_installed_apps()
  File "/usr/local/lib/python3.8/dist-packages/bench/bench.py", line 159, in get_installed_apps
    apps.remove("frappe")
ValueError: list.remove(x): x not in list
```

I have to completely remove my bench, and re-create it every time.

Here is my directory structure:
![image](https://user-images.githubusercontent.com/29856401/174144564-35b7c7ea-b979-40ef-a8ea-e5c19a062162.png)

**Comment by dj12djdjs** [comment]
I'm able to solve this by running
`./env/bin/pip3 install -e ./apps/{APP_NAME}`

for every app. I'm not sure why it looses the installed apps from the env.

Could this possibly be from switching from dev bench to production bench ?

**Comment by gavindsouza** [comment]
Fixed in #1323

**Comment by lekhnath** [comment]
@gavindsouza I don't think so
![image](https://github.com/frappe/bench/assets/4231691/2cacbace-28a9-4c71-bfa5-75fd84e58cf4)
